### PR TITLE
doc: update the pi docs according to the new changes

### PR DESCRIPTION
**Pull request by masco-enovance**

### CODE REVIEW

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by masco-enovance** [comment]
rerun functional-tests-backend-orientdb

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by masco-enovance** [comment]
run functional-tests-backend-orientdd

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

File: doc/content/use-cases/packet-injector.md [hunk 1]
```diff
@@ -12,25 +14,44 @@ To use the packet injector we need to provide the below parameters,
...
+      --type string      packet type: icmp4, icmp6, tcp4, tcp6, udp4 and udp6 (default "icmp4")
+```
+
+## Example
+```console
+$ skydive client inject-packet create --src="G.V().Has('TID', 'feae10c1-240e-48e0-4a13-c608ffd157ab')" --dst="G.V().Has('TID', 'feae10c1-240e-48e0-4a13-c608ffd15700')" --type="icmp4" --count=15
 ```
 
-### Example
+## Delete packet injector
```
Comment (lebauce) [inline doc/content/use-cases/packet-injector.md]: Delete packet injection

File: doc/content/use-cases/packet-injector.md [hunk 2]
```diff
@@ -12,25 +14,44 @@ To use the packet injector we need to provide the below parameters,
...
+Deleting the active packet injector can be done by using the `delete` sub-command with the UUID of the injector.
+
+UUID of the injector will be returend as a response of `create`
```
Comment (lebauce) [inline doc/content/use-cases/packet-injector.md]: UUID of the injection will be returned as a response of `create`

File: doc/content/use-cases/packet-injector.md [hunk 3]
```diff
@@ -12,25 +14,44 @@ To use the packet injector we need to provide the below parameters,
...
       --srcIP string     source node IP
       --srcMAC string    source node MAC
-      --type string      packet type: icmp (default "icmp")
+      --srcPort int      source port for TCP packet
+      --type string      packet type: icmp4, icmp6, tcp4, tcp6, udp4 and udp6 (default "icmp4")
+```
+
+## Example
+```console
+$ skydive client inject-packet create --src="G.V().Has('TID', 'feae10c1-240e-48e0-4a13-c608ffd157ab')" --dst="G.V().Has('TID', 'feae10c1-240e-48e0-4a13-c608ffd15700')" --type="icmp4" --count=15
```
Comment (lebauce) [inline doc/content/use-cases/packet-injector.md]: Maybe you can show the output of the command. And use the returned UUID for the `inject-packet delete` command below

**Comment by masco-enovance** [comment]
run functional-tests-backend-orientdb

**Pull request merged**